go/mysql/auth_server_static.go

```diff
@@ -116,13 +113,12 @@ func RegisterAuthServerStaticFromParams(file, str string) {
 	}
 	authServerStatic.installSignalHandlers()
 
-	authServerStatic.LastAuthRotation = time.Now()
-
 	// And register the server.
 	RegisterAuthServerImpl("static", authServerStatic)
 }
 
 func (a *AuthServerStatic) loadConfigFromParams(file, str string) {
+	// a.Entries is updated within this function, so we lock here
```

Comment:
nit: updated->accessed (need to change for all comments)

Reply:
Fixed.